lib/tasks/version.rake

```diff
@@ -0,0 +1,23 @@
+namespace :versions do
+  desc 'Sets conference_id in all pre-existing PaperTrail::Version objects'
+  task set_conference_id: :environment do
+
+    PaperTrail::Version.all.each do |version|
+      # All pre-existing versions are either of Conference or Event
+      if version.item_type == 'Conference'
+        version.conference_id = version.item_id
+        version.save
+      elsif version.item_type == 'Event'
+        event = (version.item || version.reify || version.next.reify)
```

Comment:
Even if the actual Event does not exist for some reason, the object attribute of changes has kept a record of either the conference_id (old records) or the program_id (through which we can go to conference).

That way we can ensure more (or all?) our records will get a conference_id.
So if event does not exist, search for what we want in the more hidden data ;)